src/components/manifold-plan-table/skeleton.tsx

```diff
@@ -14,35 +14,37 @@ const SkeletonLoader: FunctionalComponent = () => (
     }}
   >
     <div
-      class="mp--cell mp--cell__sticky mp--cell__bls mp--cell__al mp--cell__th mp--cell__thead mp--cell__bts"
+      class="ManifoldPlanSelector__Cell ManifoldPlanSelector__Cell--Sticky ManifoldPlanSelector__Cell--TH ManifoldPlanSelector__Cell--THead"
       data-column-first
       data-row-first
     ></div>
     {Array.from(new Array(DEFAULT_FEATURES)).map(() => (
-      <div class="mp--cell mp--cell__sticky mp--cell__bls mp--cell__al mp--cell__th">
-        <div class="mp--skeleton">                                              </div>
+      <div class="ManifoldPlanSelector__Cell ManifoldPlanSelector__Cell--Sticky ManifoldPlanSelector__Cell--TH">
+        <div class="ManifoldPlanSelector__Skeleton">
+                                                        
+        </div>
       </div>
     ))}
     <div
-      class="mp--cell mp--cell__sticky mp--cell__bls mp--cell__bbs mp--cell__al mp--cell__th"
+      class="ManifoldPlanSelector__Cell ManifoldPlanSelector__Cell--Sticky ManifoldPlanSelector__Cell--TH"
```

Comment:
here you have a `TH` but above you have a `THead`. Should these be tthe same?

Reply:
I’ve seen Terry use `<th>` tags as the left-most column in table rows before, so I’m thinking that’s the logic there. I mostly tried to make this PR about conforming to the new CSS Styleguide, not restyling this component. But I can change the name to make more sense.

Reply:
Actually I’m going to hold off on renaming; that actually is a bit cascading because it affects other things. I can revisit that in a followup.